.ipynb_checkpoints/titanic-dataset-checkpoint.ipynb

```diff
@@ -0,0 +1,934 @@
+{
```

Comment:
Os arquivos dentro dessa pasta são checkpoints do jupyter notebook que são _saves_ periódicos dos seus arquivos originais.  O jupyter mantém pra se, por algum motivo, vc perder os arquivos originais enquanto estiver codando, vc possa recuperar.
Maas são só arquivos temporários que vão se modificando conforme você vai modificando os originais, por isso não costumam ser commitados. Recomendo deletar eles da branch.
---
A título de conhecimento e pra você não precisar ficar des-selecionando esses arquivos toda vez que for fazer um commit, existe um tipo de arquivo que pode te ajudar que se chama [gitignore](https://git-scm.com/docs/gitignore). Em resumo se você adicionar esse arquivo na raiz do seu projeto, ele vai excluir da possibilidade de commitar todos os arquivos que estiverem listados nele.
Pra isso é só criar um arquivo chamado `.gitignore` (esse é o nome inteiro do arquivo é só uma extensão) e adicionar os nomes dos arquivos que vc quer excluir. Existe um [site](https://www.toptal.com/developers/gitignore) pra criar gitignores geralmente comuns. Aqui um [link de exemplo](https://www.toptal.com/developers/gitignore/api/jupyternotebooks) de um gitignore para quem costuma utilizar jupyter notebook

Reply:
Entendi, não sabia disso, obrigado =)
---
Que legal. Dei uma olhada sobre a formatação do `.gitignore` e fiz o exemplo no site que você linkou. Bem legal e útil isso. 
---
Modificações feitas.